woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -725,10 +726,7 @@ public function save() {
 
 		} catch ( \Exception $e ) {
 
-			$this->token = null;
-
-			// TODO probably this exception should be logged to the gateway log {FN 2020-02-25}
-			$saved = $token->get_id();
+			throw new SV_WC_Payment_Gateway_Exception( $e->getMessage() );
```

Comment:
```suggestion
			throw new SV_WC_Payment_Gateway_Exception( 'Could not save payment token. ' . $e->getMessage() );
```

The exception was logged as follows, so I thought it would be good to expand the message to indicate what we were trying to do:

    02-25-2020 @ 13:24:21 - Invalid or missing payment token fields.